### DISCUSSION TITLE
Please avoid suppressing discussion of topics you aren't interested in.

### DISCUSSION THREAD

**Discussion by paradoxReborn** [godotengine/godot-proposals#5374] [General Discussions]
This issue was promptly closed, blocking any further comment by anyone who is not a contributor.
https://github.com/godotengine/godot-proposals/issues/79#issue-494550939

Please consider changing settings to allow continued discussion by non-contributors or leave such issues open. Godot users such as myself cannot even add to existing reaction counts or add new reactions to posts. Some contributors have said things here that I disagree with, and others have said things I agree with. I would at least like to add my voice to this conversation.

It seems Godot's lead developer isn't interested in this discussion topic, but this isn't a valid reason for suppressing it. There are real marketing and branding concerns related to the logo, and open dialog about them can only benefit the community. This isn't specific to the logo topic, which seems to be a perennial source of contention. Open source projects thrive on open discourse and contribution. Any action that might be construed as shutting down discussion should be strongly reconsidered.

**Comment by Zireael07** [comment]
The issue was closed as it was going in circles, pretty much nowhere, proving that as one of the contributors said "it is a subjective thing"

**Reply by paradoxReborn** [reply]
> The issue was closed as it was going in circles, pretty much nowhere, proving that as one of the contributors said "it is a subjective thing"

While concerns about marketing image are subjective, an objective truth is that some professionals have concerns about the image that the Godot logo portrays. For example, I have no plans to use the Godot logo in any of my marketing materials because of its childish appearance. Instead, I'll use the old 2009 logo or the name only. This yields another objective truth: At least one developer plans not to use the logo because of its image. (There may be others we are not aware of.)

While back-and-forth about whether you find the logo appealing isn't productive, discussion about the image it creates in marketing is objectively useful... unless you consider all marketing concerns subjective and therefore useless, which is a ridiculous and indefensible position. There may be objective value in a logo that has a broader appeal to professionals, and there is certainly value in discussing this consideration.

**Reply by Zireael07** [reply]
And the adjective you just used, "childish", proves that this is very much subjective...

**Reply by paradoxReborn** [reply]
Are you asserting that subjective implies "not useful?" If that were the case, marketing (which is inherently subjective) is not useful, which is clearly not true. There's an entire professional field dedicated to marketing because it makes money.

**Reply by Zireael07** [reply]
I never said anything about "not useful" - however 1) we're already going in circles and 2) I fail to see how we can get a logo that everyone finds appealing...

**Reply by paradoxReborn** [reply]
The following objective fact remains: A developer plans not to include the Godot logo in marketing materials because of its perceived (subjective) image. That's a pretty strong statement despite being a result of someone's subjective judgement.

**Reply by Zireael07** [reply]
Yes, but how we do avoid that? No matter what logo ends up used, there is always a risk that some dev won't like the logo and won't use it in marketing...

**Reply by paradoxReborn** [reply]
You can't satisfy everyone, but you can reach a vague compromise by putting forward something that at least makes an effort to be in style with today's iconography. The lead developer seems openly opposed to any such effort, which I believe is a mistake.

Some of the icons listed [here](https://godotengine.org/press) (the monochrome ones) lean in the right direction, but they aren't quite there yet. If you take one of the monochrome logos, remove the pupil and change the eyes to a rounded rectangle or "pill" shape, you get something that would look fine even on the splash screen of a AAA title. Those are the sorts of changes that should be discussed, ideally with informed input from marketing professionals.

**Reply by miv391** [reply]
If you make an AAA title, it really doesn't matter at all what kind of logos the game's splash screen contains. I would suggest first making the AAA title, then forking Godot and changing the logo to whatever you want.

**Reply by theraot** [reply]
It might be worth pointing out that you don't need to fork Godot to change the logo. Because Godot does not forces you to use it, you can replace it (change the icon and the splash screen) on your projects with anything, including your own version of the logo. And yes, you are free to make your own interpretation of Godot's logo. If you are modifying the existing one as your starting point, know it is under a creative commons license. @paradoxReborn @miv391

**Comment by PoisonousGame** [comment]
The current logo does seem childish, and the first impression it gives is toy and unprofessional.

**Reply by paradoxReborn** [reply]
It's the eyes. They look like something that would be painted on a child's toy. The robot head works fine as a logo, but it needs less cartoonish eyes.

**Reply by Zireael07** [reply]
Now that's something concrete the team could work with...

**Reply by miv391** [reply]
Ferrari's logo has many colors and it has a horse. It must be drawn by a child. Would you advice Ferrari to change their logo to something more professional? Maybe just a "ferrari" text in black Arial. I don't know what would be more professional.

**Comment by Mickeon** [comment]
By all means, open a discussion on the topic again if it does interest you and there's no concrete solution on the problem. That's the point of discussions, in fact.

**Comment by wyvernbw** [comment]
Fixed it for you

![cool godot](https://user-images.githubusercontent.com/50150016/198406726-0f2abbb7-7e4b-4b27-861a-f2233a58a416.png)

**Comment by dalexeev** [comment]
> The following objective fact remains: A developer plans not to include the Godot logo in marketing materials because of its perceived (subjective) image.

Tux, the Linux Mascot (which many perceive and use as the Linux logo) is also quite funny and frivolous, but no one perceives Linux as an unprofessional project and is not shy about using it on marketing materials.

![](https://user-images.githubusercontent.com/47700418/198507861-4f1f3a35-b54f-4847-8711-8bf3d6f6891b.png)

However, I am very saddened by the fact that people are worried that the Godot logo may come across as unprofessional, this is a very wrong criterion for judging the quality of a product and shows how much people still think in terms of stereotypes.

I would like to say that it's great that the key developers are not devoid of self-irony and humor, not worried that Godot can be perceived as childish, funny and unprofessional, despite the fact that this is complete nonsense. Freedom from stereotypes, insubordination to the soulless fashion imposed by corporations, and so on.

But I must admit that these fears are not unfounded, and it is not always possible to go against the crowd and fully preserve your individuality and originality. And if the Godot logo really repels a certain percentage of developers and businesses, then this is not a very good fact. It would be much better to try to find some kind of compromise, rather than declare adamant refusal.

I think it would be possible to slightly change the contours and/or colors to get an option that would be more "professional", but at the same time not lose recognition, identity and emotions. I don't think we should move to a monochrome version or something like that from the closed proposal. This may be a small change, similar to what happened when moving from version 2 to version 3. But this small change may be enough to improve the situation without confusing existing users.

![](https://user-images.githubusercontent.com/47700418/198521190-f51fdebe-92db-496d-a98a-0d658215e5f6.png)

Maybe we need a poll?

**Reply by Zireael07** [reply]
I remember several ideas about changing colors/contours to e.g. differentiate beta and stable builds. Can't find the issue # though.

**Reply by Mickeon** [reply]
This progression, and how users proposed the Godot logo to look like, made me think of how United States videogame box-arts used to be.
![image](https://user-images.githubusercontent.com/66727710/198538506-3d498e63-1021-4b3c-9f8a-511966288bc2.png)

**Comment by Zireael07** [comment]
Can one of the mods rename the title to something more relevant e.g. "Change Godot logo"?

**Reply by paradoxReborn** [reply]
Please do NOT do that. This thread is not intended to be a discussion of the logo. I started this thread primarily about the Godot project's github settings, which only allow contributors to add comments or reactions to closed issues. Either the settings or the policies regarding what topics to close need to be changed.

**Reply by paradoxReborn** [reply]
There are several posts here about the logo, but that wasn't the reason for creating this discussion thread. Please do not change the title.

**Comment by seocwen** [comment]
Coders... Sometimes they act like they're the only ones who do any work. Graphic design is a thing, and no, not all logos are created equal. Nor is every critique of even quality. Art teams don't just do whatever they want (or whatever the non-artists think they want). A good logo takes a lot of thought, and to a designer, involves a considerable amount of objective criteria besides the ability to synthesize different stakeholder opinions.

We give users our games to test because game quality _isn't_ 100% subjective, and because none of us can know everything about what it takes to make a good game, or even what our games feel like. Granted, a good developer also needs to know when their users are complaining about symptoms rather than causes and tackle the deeper issues their playtesters can't identify. Graphic design is pretty similar to game design in this respect.

I'm really more of an artist than a designer, so I won't wade in, but no branding is so perfect that it doesn't need an update now and then. Every large media organizer should have a full-time designer, and not just for its logo. And no, a moonlighting coder isn't going to cut it--get an RGD with professional qualifications at minimum.
